requirements.md

```diff
@@ -16,29 +16,46 @@ See also: [SPC Scope](scope.md) for definitions and more information.
 ## Web Context Support
 
 * It must possible to invoke SPC from an iframe. See [issue 68](https://github.com/w3c/secure-payment-confirmation/issues/68).
+
+Q: I understand that woudl also require PR API to be used from within the IFrame. Is that right?
+
 * It must possible to invoke SPC from within Payment Request API.
+
+Q: Shouldn't that be rather "It must be possible to invoke SPC via Payment Request API"? I understand the intention is to use the PR API as a way to invoke the SCP.
```

Comment:
One idea had been to create something like an authenticate() method. In a world where show() is used to get instrument information, it could be followed by authenticate(). Or, one could call authenticate() without calling show() if the instrument is already known.

Reply:
@ianbjacobs: interesting idea. I'm not very keen on using the payment method identifier literal to trigger the SPC, so perhaps the explicit method would be an interesting alternative.

Reply:
I'm going to resolve this piece of the discussion since we are moving the document in a way that is independent of PR API and WebAuthn (in terms of capability). The document now says "It must be possible to call SPC from a Web site or a payment handler. "